contracts/src/libraries/SafeCast.sol

```diff
@@ -2,10 +2,13 @@
 pragma solidity 0.8.20;
 
 import { IHyperdrive } from "../interfaces/IHyperdrive.sol";
+import { Errors } from "./Errors.sol";
```

Comment:
I know this isn't on you, but it'd be good to get tests for these methods bc rn there are 0.

Reply:
maybe create an issue. this is something that cash, matt or john could do

Reply:
I opened an issue. I'll start adding to it over the course of this week.